dataservice/api/family_relationship/schemas.py

```diff
@@ -0,0 +1,27 @@
+from marshmallow_sqlalchemy import field_for
+
+from dataservice.api.family_relationship.models import FamilyRelationship
+from dataservice.api.common.schemas import BaseSchema
+from dataservice.extensions import ma
+
+
+class FamilyRelationshipSchema(BaseSchema):
+    participant_id = field_for(FamilyRelationship, 'participant_id',
+                               required=True,
+                               load_only=True, example='PT_B048J5')
+    relative_id = field_for(FamilyRelationship, 'relative_id',
+                            required=True,
+                            load_only=True, example='PT_B048J6')
+
+    class Meta(BaseSchema.Meta):
+        model = FamilyRelationship
+        resource_url = 'api.family_relationships'
+        collection_url = 'api.family_relationships_list'
+        exclude = ('relative', 'participant')
+
+    _links = ma.Hyperlinks({
+        'self': ma.URLFor(Meta.resource_url, kf_id='<kf_id>'),
+        'collection': ma.URLFor(Meta.collection_url),
+        'participant': ma.URLFor('api.participants', kf_id='<participant_id>'),
```

Comment:
I know we've discussed doing this in #104, but do we want to do it to all resources at once to stay consistent? How do we track?

Reply:
Welp, I think I might have added these types of links for the other resources I did...

Reply:
oh well, we'll just need to make sure the others get done.